unscented_kalman_filter/include/unscented_kalman_filter/ukf_filter.h

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (C) 2019 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
```

Comment:
Talked with John, you probably want to keep the MIT license file in those two library files since you are making modification on the Udacity course project. And be sure to include a change notice for all the enhancement you have in this library.

Reply:
Marked as resolve for duplicate comments.